src/forklift/__main__.py

```diff
@@ -45,7 +45,7 @@
 from os.path import join
 
 log_location = join(abspath(dirname(__file__)), '..', 'forklift-garage', 'forklift.log')
-detailed_formatter = logging.Formatter(fmt='%(levelname).4s %(asctime)s %(module)10s:%(lineno)5s %(message)s',
+detailed_formatter = logging.Formatter(fmt='%(levelname)7s %(asctime)s %(module)10s:%(lineno)5s %(message)s',
```

Comment:
before i had the log truncated to 4 characters because i thought it looked nice but it would only print things like 
```
DEBU
INFO
WARN
ERRO
```

Warning is the longest log level so I gave it 7 characters. There's a nice language called log for atom that will color code these values. That was the impetus for this change.